tests/checks/integration/test_process.py

```diff
@@ -233,8 +250,13 @@ def deny_name(obj):
         self.run_check(config, mocks={'get_pagefault_stats': noop_get_pagefault_stats})
 
     def mock_find_pids(self, name, search_string, exact_match=True, ignore_ad=True,
-                       refresh_ad_cache=True):
-        idx = search_string[0].split('_')[1]
+                       refresh_ad_cache=True, pid=None, pid_file=None):
```

Comment:
This change is not needed, `find_pids` is not called when `pid` or `pid_file` are there, only when `search_string` is specified.